e2e/tests/interchain_accounts/params_test.go

```diff
@@ -0,0 +1,92 @@
+package interchain_accounts
+
+import (
+	"context"
+	"testing"
+
+	govtypes "github.com/cosmos/cosmos-sdk/x/gov/types"
+	paramsproposaltypes "github.com/cosmos/cosmos-sdk/x/params/types/proposal"
+
+	controllertypes "github.com/cosmos/ibc-go/v7/modules/apps/27-interchain-accounts/controller/types"
+	icatypes "github.com/cosmos/ibc-go/v7/modules/apps/27-interchain-accounts/types"
+	ibctesting "github.com/cosmos/ibc-go/v7/testing"
+
+	"github.com/cosmos/ibc-go/e2e/testsuite"
+	"github.com/cosmos/ibc-go/e2e/testvalues"
+	"github.com/strangelove-ventures/interchaintest/v7/ibc"
+	"github.com/stretchr/testify/suite"
+)
+
+func TestInterchainAccountsParamsTestSuite(t *testing.T) {
+	suite.Run(t, new(InterchainAccountsParamsTestSuite))
+}
+
+type InterchainAccountsParamsTestSuite struct {
+	testsuite.E2ETestSuite
+}
+
+// QueryControllerParams queries the params for the controller
+func (s *InterchainAccountsParamsTestSuite) QueryControllerParams(ctx context.Context, chain ibc.Chain) controllertypes.Params {
+	queryClient := s.GetChainGRCPClients(chain).ICAControllerQueryClient
+	res, err := queryClient.Params(ctx, &controllertypes.QueryParamsRequest{})
+	s.Require().NoError(err)
+
+	return *res.Params
+}
+
+func (s *InterchainAccountsParamsTestSuite) TestControllerParams() {
```

Comment:
To make the function naming similar to the params tests we have for transfer, for example.

```suggestion
func (s *InterchainAccountsParamsTestSuite) TestControllerEnabledParam() {
```